nemo/collections/asr/models/rnnt_models.py

```diff
@@ -91,6 +91,11 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         else:
             self.spec_augmentation = None
 
+        if loss_name == 'tdt':
```

Comment:
This part should be closer to the loss definition above. Colocation of similar things

Reply:
done